ParabolInc/parabol
# feat(jira-server): Allow searching for issues in Sprint Poker

#6097
<img width="1480" alt="image" src="https://user-images.githubusercontent.com/466991/164016514-f7432ed3-54f5-4a89-9cde-d275a186d906.png">

**How to test:**
* connect jira server
* create poker meeting
* go to scope on jira server tab
* Check that search, JQL search, project filters works
